pyomo/contrib/gdpopt/util.py

```diff
@@ -206,8 +206,7 @@ def copy_var_list_values(from_list, to_list, config,
             rounded_val = int(round(var_val))
             # Check to see if this is just a tolerance issue
             if ignore_integrality \
-                and ('is not in domain Binary' in err_msg
-                     or 'is not in domain Integers' in err_msg):
+                    and not v_to.is_continuous():
```

Comment:
Would you mind updating the other entries with the equivalent checks? `is_binary()` and `is_integer()` below.

Reply:
You were right. Just to clarify, instead of implementing the condition with `is_integer` we did it with `not is_continuous`. We are still pushing to the rounded value.